src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ProjectDebugPropertyPage.xaml

```diff
@@ -26,7 +26,15 @@
 
   <StringProperty Name="CommandLineArguments"
                   DisplayName="Command line arguments"
-                  Description="Command line arguments to pass to the executable." />
+                  Description="Command line arguments to pass to the executable.">
+    <StringProperty.ValueEditors>
+     <ValueEditor EditorType="MultiLineString">
+      <ValueEditor.Metadata>
+        <NameValuePair Name="UseMonospaceFont" Value="True" />
+      </ValueEditor.Metadata>
+    </ValueEditor>
+    </StringProperty.ValueEditors>
+  </StringProperty>
```

Comment:
super minor nit: the indentation is a bit inconsistent here.

Reply:
Fixed the indentation. Sorry!